gradoop-flink/src/main/java/org/gradoop/flink/util/GradoopFlinkConfig.java

```diff
@@ -46,7 +46,7 @@
   /**
    * Creates instances of {@link LogicalGraph}
    */
-  private final BaseGraphFactory<GraphHead, Vertex, Edge, LogicalGraph> logicalGraphFactory;
+  private final BaseGraphFactory<GraphHead, Vertex, Edge, LogicalGraph, GraphCollection> graphFac;
```

Comment:
Imo write the name out `graphFactory`. The collection below also has a line break caused by the var name.